src/Business Foundation/App/NoSeries/src/Single/NoSeriesStatelessImpl.Codeunit.al

```diff
@@ -17,6 +17,9 @@ codeunit 306 "No. Series - Stateless Impl." implements "No. Series - Single"
     var
         CannotAssignNumbersGreaterThanErr: Label 'You cannot assign numbers greater than %1 from the number series %2. No. assigned: %3', Comment = '%1=Last No.,%2=No. Series Code, %3=the new no.';
         WarnNoSeriesRunningOutMsg: Label 'The No. Series %1 is soon running out. The current number is %2 and the last allowed number of the sequence is %3.', Comment = '%1=No. Series code,%2=Current No.,%3=Last No. of the sequence';
+        NoSeriesLbl: Label 'No. Series', Locked = true;
+        NoSeriesOutOfRangeErr: Label 'The last number used is outside range.', Locked = true;
```

Comment:
Should also just be a Lbl ... (or Tok) ... it's not used in an error, it is just sent to telemetry...